File: BAEKJOON/01003/solution.py
#https://www.acmicpc.net/problem/1003

count = {0:[1, 0], 1:[0, 1]}

inputs = [] # test case T
for i in range(int(input())):
    inputs.append(int(input())) # add input N to list

for j in range(2, max(inputs) + 1):
    count[0].append(count[0][j - 1]  + count[0][j - 2])
    count[1].append(count[1][j - 1]  + count[1][j - 2])

for x in inputs:
    print(count[0][x], count[1][x])

File: BAEKJOON/01003/solution.c
// https://www.acmicpc.net/problem/1904
//20210216

#include<stdio.h>

int main()
{
	int arr[41] = { 0, 1, };
	int t, i;
	scanf("%d", &t);
	for (i = 2; i < 41; i++) arr[i] = arr[i - 1] + arr[i - 2];

	while (t-- > 0)
    {
		scanf("%d", &i);
		switch (i)
        {
		case 0:
			puts("1 0");
			break;
		case 1:
			puts("0 1");
			break;
		default:
			printf("%d %d\n", arr[i - 1], arr[i]);
		}
	}

	return 0;
}
